src/objects/Funding_Request__c.object

```diff
@@ -197,6 +197,14 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>Identifier__c</fullName>
+        <displayFormat>FR-{000000}</displayFormat>
```

Comment:
Quick note -- Disbursement's auto-number is currently using 5 digits rather than 6 if we wanted to be consistent.